app.js

```diff
@@ -0,0 +1,157 @@
+//jshint esversion: 6
+const express = require("express");
+const bodyParser = require("body-parser");
+const mongoose = require("mongoose");
+const redisClient = require("redis").createClient;
+const redis = redisClient(6379, 'redis-server');
+
+const app = express();
+
+app.set('view engine', 'ejs');
+app.use(bodyParser.urlencoded({
+  extended: true
+}));
+app.use(express.static("public"));
+//let studentMap = new Map();
```

Comment:
When finally giving code for review, try to remove possible comments from all the places